### PR TITLE
fix not valid markdown, and prettify them

### PR DESCRIPTION
**Pull request by dimaMachina**

### CODE REVIEW

File: packages/plugins/response-cache-redis/README.md [hunk 1]
```diff
@@ -37,53 +35,53 @@ import Redis from 'ioredis';
...
   port: '30652',
-  password: '1234567890',
-});
+  password: '1234567890'
+})
 
-// or, you can also specify connection options as a redis:// URL or rediss:// URL when using TLS encryption
-const redis = new Redis(("rediss://:1234567890@my-redis-db.example.com':30652");
+// or, you can also specify connection options as a redis:// URL or redis:// URL when using TLS encryption
+const redis = new Redis('redis://:1234567890@my-redis-db.example.com:30652')
```
Comment (dimaMachina) [inline packages/plugins/response-cache-redis/README.md]:
```suggestion
// or, you can also specify connection options as a redis:// URL or rediss:// URL when using TLS encryption
const redis = new Redis('rediss://:1234567890@my-redis-db.example.com:30652')
```

File: packages/plugins/response-cache-redis/README.md [hunk 2]
```diff
@@ -37,53 +35,53 @@ import Redis from 'ioredis';
...
+import { envelop } from '@envelop/core'
+import { useResponseCache } from '@envelop/response-cache'
+import { createRedisCache } from '@envelop/response-cache-redis'
 
-import { emitter } from './eventEmitter';
+import { emitter } from './eventEmitter'
 
 // we create our cache instance, which allows calling all methods on it
-const redis = new Redis(("rediss://:1234567890@my-redis-db.example.com':30652");
+const redis = new Redis('redis://:1234567890@my-redis-db.example.com:30652')
```
Comment (dimaMachina) [inline packages/plugins/response-cache-redis/README.md]:
```suggestion
const redis = new Redis('rediss://:1234567890@my-redis-db.example.com:30652')
```

File: website/docs/guides/adding-a-graphql-response-cache.mdx [hunk 3]
```diff
@@ -328,28 +328,27 @@ Then, with that instance of the Redis Cache setup, provide it to the `useRespons
...
 const redis = new Redis({
   host: 'my-redis-db.example.com',
   port: '30652',
-  password: '1234567890',
-});
+  password: '1234567890'
+})
 
-const redis = new Redis(("rediss://:1234567890@my-redis-db.example.com':30652");
+const redis = new Redis('redis://:1234567890@my-redis-db.example.com:30652')
```
Comment (dimaMachina) [inline website/docs/guides/adding-a-graphql-response-cache.mdx]:
```suggestion
const redis = new Redis('rediss://:1234567890@my-redis-db.example.com:30652')
```

File: packages/core/docs/use-extend-context.md [hunk 4]
```diff
@@ -3,17 +3,19 @@
...
+import { buildSchema } from 'graphql'
 
 const getEnveloped = envelop({
   plugins: [
-    useExtendContext(async (contextSoFar) => {
+    useExtendContext(async contextSoFar => {
       return {
-        myCustomField: { ... }
+        myCustomField: {
+          /* ... */
```
Comment (dimaMachina) [inline packages/core/docs/use-extend-context.md:14]: `...` need to put in comment to make valid syntax

File: packages/plugins/execute-subscription-event/README.md [hunk 5]
```diff
@@ -7,35 +7,35 @@ Utilities for hooking into the [ExecuteSubscriptionEvent](<https://spec.graphql.
...
-import { useContextValuePerExecuteSubscriptionEvent } from '@envelop/execute-subscription-event';
-import { createContext, createDataLoaders } from './context';
+import { envelop } from '@envelop/core'
+import { useContextValuePerExecuteSubscriptionEvent } from '@envelop/execute-subscription-event'
+import { createContext, createDataLoaders } from './context'
 
 const getEnveloped = envelop({
   plugins: [
-    useContext(() => createContext())
+    useContext(() => createContext()),
```
Comment (dimaMachina) [inline packages/plugins/execute-subscription-event/README.md:16]: missing comma to make valid syntax

File: packages/plugins/execute-subscription-event/README.md [hunk 6]
```diff
@@ -7,35 +7,35 @@ Utilities for hooking into the [ExecuteSubscriptionEvent](<https://spec.graphql.
...
```
Comment (dimaMachina) [inline packages/plugins/execute-subscription-event/README.md:38]: missing comma

File: website/docs/guides/resolving-subscription-data-loader-caching-issues.mdx [hunk 7]
```diff
@@ -19,53 +19,54 @@ This is a known issue that still has no solution in GraphQL.js core, although, [
...
 const GraphQLSubscriptionType = new GraphQLObjectType({
   name: 'Subscription',
   fields: {
     something: {
       type: GraphQLNonNull(GraphQLUserListUpdate),
-      resolve: (_, __, context) => pipe(
-          context.pubSub.subscribe("userUpdate")
+      resolve: (_, __, context) =>
+        pipe(
+          context.pubSub.subscribe('userUpdate'),
```
Comment (dimaMachina) [inline website/docs/guides/resolving-subscription-data-loader-caching-issues.mdx:40]: missing comma

File: website/docs/guides/resolving-subscription-data-loader-caching-issues.mdx [hunk 8]
```diff
@@ -19,53 +19,54 @@ This is a known issue that still has no solution in GraphQL.js core, although, [
...
-import { createContext, createDataLoaders } from './context';
+import { envelop } from '@envelop/core'
+import { useContextValuePerExecuteSubscriptionEvent } from '@envelop/execute-subscription-event'
+import { createContext, createDataLoaders } from './context'
 
 const getEnveloped = envelop({
   plugins: [
     // ... other plugins ...
-    useContext(() => createContext())
+    useContext(() => createContext()),
```
Comment (dimaMachina) [inline website/docs/guides/resolving-subscription-data-loader-caching-issues.mdx:61]: missing comma

File: website/docs/plugins/lifecycle.mdx [hunk 9]
```diff
@@ -55,15 +55,15 @@ So for example, if you wish to make some data from `parse` stage available for y
 
 ```ts
 const myPlugin = {
-  onParse({ extendContext }) => {
+  onParse({ extendContext }) {
```
Comment (dimaMachina) [inline website/docs/plugins/lifecycle.mdx:58]: no need to arrow, invalid syntax, it's object property shorthand

**Review by n1ru4l — APPROVED** [review]

**Pull request merged**